### ISSUE TITLE
Problem in running Lazy config for Detic_ViLD training 

### ISSUE THREAD

**Issue by Uzair-Khattak** [facebookresearch/Detic#51]
Hi @xingyizhou 

Thank you for sharing the great work. 

I am trying to reproduce the Detic results on LVIS.
Firstly Box Supervised ViLD baseline model has been successfully trained. 

Now using its weights, I am trying to run Detic configuration using [Detic_ViLD_200e.py](https://github.com/facebookresearch/Detic/blob/main/configs/Detic_ViLD_200e.py) config file, but after running the  `lazy_train_net.py`, it prints initial logs and get stuck when iteration 0 begins to start.
I am attaching screenshots below, after the last warning, the training does not get started, while the utilization from all GPUs instantly reaches 100% and get stuck on 100%
![image](https://user-images.githubusercontent.com/61704085/167276579-a98d8f9d-de5d-451f-8f18-c361b7715abd.png)
![image](https://user-images.githubusercontent.com/61704085/167276662-dd44ec6f-345b-4bb5-870e-d6692b70df59.png)
Even after quitting the python run, the memory usage still get stuck to 100%, I have to manually kill the python process in-order to reset the GPU utilization. 

Can you please help me in running the Detic training for LVIS.
I am using the same environment, workstation etc on which the Box Supervised ViLD baseline is trained. 

Waiting for your kind response. 
Thank you.

**Comment by xingyizhou** [comment]
Hi,

  Detic requires (much) more CPU memory due to the use of ImageNet data. A simple workaround is to reduce the [number of data preprocessing workers](https://github.com/facebookresearch/Detic/blob/main/configs/Detic_ViLD_200e.py#L131). If this does not work, try to reduce the batchsize (and increase the training iterations and decrease the learning rate following the linear learning rate rule).

**Comment by Uzair-Khattak** [comment]
Thank you for your immediate reply. I will check this out and let you know.
kind regards.

**Comment by Uzair-Khattak** [comment]
Hi @xingyizhou, 

Thank you once again.
I have tried to make the suggested changes but the code keeps on getting stuck. Even with `num_workers=1` and reduced batch size ([8,8] for 8 GPUs), the code does not proceed after printing the iteration 0 on the console.

I have tried it on both A100 and V100 GPUs.

With the given model weights for Detic_ViLD [Detic_ViLD_200e.pth](https://drive.google.com/file/d/178T7vEbDRW5B82kZiQIPxOrsz8Mqb-B2/view), we are able to reproduce the evaluation results (screenshot attached below), but when using the same script for training, it hangs.. 
![image](https://user-images.githubusercontent.com/61704085/167294575-08c39de4-69f2-4360-86d9-bebf123a7ccc.png)

During debugging of the code, the code hangs when the input images are passed through the backbone in this particular [line](https://github.com/facebookresearch/Detic/blob/f633b7869f87a33fe9ee90b824301ef5e98f0fbf/detic/modeling/meta_arch/custom_rcnn.py#L137).

I am not sure if the problem is in loading the data as I am able to print out and view the `images ` shape and tensor values during debugging, when It is passed through the backbone, it hangs there.

If possible for you, can you please try running and verifying the Detic_ViLD training and [Detic_ViLD_200e.py](https://github.com/facebookresearch/Detic/blob/main/configs/Detic_ViLD_200e.py) config file. 

For the environment, we are using PyTorch version `1.10.0+cu113`.


Kind regards.

**Comment by xingyizhou** [comment]
Hi,

  I verified and confirmed the code can run under cuda 11.1 and torch 1.8.1-lst. Can you try this environment? It seems other users [here](https://github.com/facebookresearch/Detic/issues/32) can start the training as well.

Best,
Xingyi

**Comment by Uzair-Khattak** [comment]
Thank you for you response. I will check it using the above mentioned PyTorch settings. 

kind regards,
Muhammad Uzair

**Comment by wangzhengchao** [comment]
Hi, @muzairkhattak 
I'm learning to train detic on LVIS.   The command  is python train_net.py --num-gpus 8 --config-file configs/Base-C2_L_R5021k_640b64_4x.yaml .  But I failed.  Firstly, I downlowd detic project and install it .  Second i prepare data.  Could you teach me  how to train  this  detic project ?
